pkg/xds/conv/convertxds_test.go

```diff
@@ -738,7 +822,7 @@ func Test_convertPerRouteConfig(t *testing.T) {
 		if !(rawFault.Abort.Status == 500 &&
 			rawFault.Abort.Percent == 100 &&
 			rawFault.Delay.Delay == time.Second &&
-			rawFault.Delay.Percent == 100 &&
```

Comment:
`rawFault.Delay.Percent` and `rawFault.Delay.DelayInjectConfig.Percent`  are equivalent.

Reply:
ok